Buildings/package.mo

```diff
@@ -196,6 +196,33 @@ its class name ends with the string <code>Beta</code>.
     <b style=\"color:blue\">non-backward compatible</b> way:
     </p>
     <table class=\"releaseTable\" summary=\"summary\" border=\"1\" cellspacing=0 cellpadding=2 style=\"border-collapse:collapse;\">
+     <tr><td colspan=\"2\"><b>Buildings.Controls.OBC.ASHRAE.G36_PR1</b>
+        </td>
+    </tr>
+    <tr><td valign=\"top\">Buildings.Controls.OBC.ASHRAE.G36_PR1.AHUs.SingleZone.VAV.Controller
+        </td>
+        <td valign=\"top\">Improved the controller implementation. <br/>
```

Comment:
Be more specific than simply "Improved the controller implementation." Write what you changed it; that it is improved is clear, otherwise there would not be a change to the model.

Also, remove the trailing space after the period.